parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_hub_wococo_config.rs

```diff
@@ -51,6 +51,14 @@ parameter_types! {
 	pub RococoGlobalConsensusNetwork: NetworkId = NetworkId::Rococo;
 	pub ActiveOutboundLanesToBridgeHubRococo: &'static [bp_messages::LaneId] = &[DEFAULT_XCM_LANE_TO_BRIDGE_HUB_ROCOCO];
 	pub PriorityBoostPerMessage: u64 = 921_900_294;
+
+	pub FromAssetHubWococoToAssetHubRococoRoute: SenderAndLane = SenderAndLane::new(
+		ParentThen(X1(Parachain(1000))).into(),
+		DEFAULT_XCM_LANE_TO_BRIDGE_HUB_ROCOCO,
+	);
+
+	pub CongestedMessage: Xcm<()> = unimplemented!("TODO: not supported yet!");
```

Comment:
Unrelated q dumb - what's the state of the R<>W bridge? Does it have all latest changes (i.e. using `pallet-xcm` instead of our own dedicated pallet)? I haven't been testing dynamic fees there and maybe some manual tests are required before we'll do a runtime upgrade there

Reply:
well, rococo/wococo is bit outdated,
I have TODO for Ro/Wo,
but I am waiting for monorepo, and testing runtimes should stay in Cumulus, so probably I will create real `asset-hub-rockmine` runtime for Rococo/Wococo, because now it is used asset-hub-kusama and with hardcoded bridging config, it is not possible to add there support for Rococo/Wococo bridging

Reply:
https://github.com/paritytech/cumulus/issues/3020